cypress/integration/26-register_publication/1693-creator_selects_reference_type_report_and_selects_subtype/subtypes_report.js

```diff
@@ -0,0 +1,70 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import { REPORT_FIELDS, REPORT_SUBTYPES } from '../../../support/data_testid_constants';
+
+const filename = 'example.txt';
+
+// Start common steps for
+// Scenario: Creator sees fields for Reference subtypes for "Report"
+// and
+// Scenario: Creator sees that fields are validated for Reference subtypes for "Report"
+
+Given('Creator begins registering a Publication in the Wizard', () => {
+  cy.login(USER_WITH_AUTHOR);
+  cy.startRegistrationWithFile(filename);
+  cy.get('[data-testid=registration-file-start-button]').click({ force: true });
+});
+When('they navigate to the Reference tab', () => {
+  cy.get('[data-testid=nav-tabpanel-reference').click({ force: true });
+});
+And('they select the Reference type "Report"', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');

```

Comment:
kom til å tenke på at når datamodellen en dag blir "utflatet" kommer vi ikke til å ha begrepet "publication-context" lenger, og alle disse data-testidene (++) kommer til å endres, så det hadde kanskje vært greit å legge  alle `data-testid`er i ei egen fil, så kunne vi bare endra en plass i stedet for overalt. kan være noe å vurdere i hvert fall